test/Interop/Cxx/function/Inputs/module.modulemap

```diff
@@ -0,0 +1,4 @@
+module EmitCalledFunction {
+  header "EmitCalledFunction.h"
```

Comment:
(nit) Should we keep to a consistent naming convention for files (lowercase with dashes)?

Reply:
Good point -- done.